### DISCUSSION TITLE
Windows 7

### DISCUSSION THREAD

**Discussion by GitLang** [VirtualDrivers/Virtual-Display-Driver#183] [Q&A] [answered]
Is there even a remote chance of a Windows 7 compatible driver please?  (don't ask...)

**Comment by bud3699** [comment, marked as answer]
Nope, I'm afraid not. Although don't give up hope too soon, although I don't believe there is any umdf drivers out there for windows 7 displays I do believe kernal level drivers will be.

I know you said don't ask, but curiosity is taking over, I understand if you do not wish to disclose it. (I don't blame you if you want to stay on windows 7 either, back before windows shoved bloatware down our throats)
